### ISSUE TITLE
Implement proper logging by default

### ISSUE THREAD

**Issue by Maykeye** [WayfireWM/wayfire#1985]
>log of Wayfire ([printed to stdout by default](https://github.com/WayfireWM/wayfire/wiki/Tutorial)).

* Stdout is not meant for log. /var/log is (or whatever its  XDG equivalent)
* Once launched from sddm,  stdout is not accessible. /var/log is (or whatever its  XDG equivalent).
* If machine is rebooted stdout does not exist. /var/log is (or whatever its  XDG equivalent)

Signed: someone whose WM crashed and I can't look anything because nothing was logged.

**Comment by ammen99** [comment]
> * Stdout is not meant for log. /var/log is (or whatever its  XDG equivalent)

Many applications including other wayland compositors print their logs to stdout.

> * Once launched from sddm,  stdout is not accessible. /var/log is (or whatever its  XDG equivalent).

Sounds like an SDDM/session configuration problem. You can redirect Wayfire's output to a file, for example the `startwayfire` script (provided by wf-install if you install Wayfire in this way) uses a log file in `~/.local/share/wayfire`


TLDR: not planned, Wayfire is just the compositor, if you want full session integration with everything else, write a wrapper script for that.
